src/cdk/constructs/CheckExpirationLambdaEdge.ts

```diff
@@ -26,17 +23,15 @@ export class CheckExpirationLambdaEdge extends Construct {
   public readonly lambdaEdge: cloudfront.experimental.EdgeFunction
 
   constructor(scope: Construct, id: string, props: CheckExpirationLambdaEdgeProps) {
-    const { bucketName, bucketRegion, ebAppDomain, nodejs, buildOutputPath, cacheConfig } = props
+    const { nodejs, buildOutputPath, cacheConfig, renderWorkerQueueUrl } = props
     super(scope, id)
 
     const nodeJSEnvironment = NodeJSEnvironmentMapping[nodejs ?? ''] ?? NodeJSEnvironmentMapping['20']
     const name = 'checkExpiration'
 
     buildLambda(name, buildOutputPath, {
       define: {
-        'process.env.S3_BUCKET': JSON.stringify(bucketName),
```

Comment:
not in use, were copied from another lambda. 
Actual lambda code updates will go in separate PR